rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Hello, kindly make sure that Getters and Setters for @book and @person instance variables are present

```sh
attr_accessor :date, :book, :person
```